lms/djangoapps/verify_student/services.py

```diff
@@ -112,3 +117,32 @@ def get_attempts(self, user_id, course_id, related_assessment_location):
         """
         course_key = CourseKey.from_string(course_id)
         return VerificationStatus.get_user_attempts(user_id, course_key, related_assessment_location)
+
+    def _is_enrolled_as_verified(self, user_id, course_key):
+        """
+        Check whether the user is enrolled in a verified track.
+
+        Arguments:
+            user_id (str): Identifier for the user.
+            course_key (CourseKey): Identifier for the course.
+
+        Returns: bool
+
+        """
+        try:
+            user = User.objects.get(id=user_id)
+        except User.DoesNotExist:
+            log.warning(
+                (
+                    "Could not find user with ID %s while checking whether "
+                    "the user can submit an in-course reverification"
+                ), user_id
+            )
+            return False
```

Comment:
curious why we're handling this one gracefully.  not being able to find the user seems like it would indicate a pretty significant error, either in code or configuration - wouldn't it be better not to squelch such errors?